<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plotter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>911</width>
      <height>651</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="plot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <attribute name="title">
      <string>графики</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ch1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>911</width>
         <height>555</height>
        </rect>
       </property>
       <attribute name="label">
        <string>канал 1</string>
       </attribute>
       <widget class="QCustomPlot" name="p_ch1_lo1" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>891</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="p_ch1_lo2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>891</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>520</y>
          <width>512</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f0=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_f0_1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Hz,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>P=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_p_1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>dBm,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Fstart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fstart_comb1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Fend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fend_comb1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>kHz</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>240</y>
          <width>186</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>x=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="edit_x1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>y=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="edit_y1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="ch2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>911</width>
         <height>555</height>
        </rect>
       </property>
       <attribute name="label">
        <string>канал 2</string>
       </attribute>
       <widget class="QCustomPlot" name="p_ch2_lo2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>891</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="p_ch2_lo1" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>891</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>520</y>
          <width>550</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f0=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_f0_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Hz,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>P=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_p_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>dBm,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Fstart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fstart_comb2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fend_comb2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>kHz</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>230</y>
          <width>186</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>x=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="edit_x2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>y=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="edit_y2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>настройки</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>391</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>UART</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>40</y>
         <width>361</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="port_edit">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/dev/ttyUSB0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>baudrate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="baud_edit">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;115200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>parity</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="parity_comb"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>stopbits</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stp_bts_comb"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>bytesizes</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="byte_comb"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="save_uart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>сохранить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>1</y>
      <width>681</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cont_cb">
       <property name="text">
        <string>непрерывные измерения</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlotter">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>очистить</string>
     </property>
     <widget class="QMenu" name="menu_1">
      <property name="title">
       <string>канал1</string>
      </property>
      <addaction name="action_clean1_today"/>
      <addaction name="action_clean1_all"/>
     </widget>
     <widget class="QMenu" name="menu_3">
      <property name="title">
       <string>канал2</string>
      </property>
      <addaction name="action_clean2_today"/>
      <addaction name="action_clean2_all"/>
     </widget>
     <widget class="QMenu" name="menu_1_2">
      <property name="title">
       <string>канал 1 и 2</string>
      </property>
      <addaction name="action_clean1_2_today"/>
      <addaction name="action_clean1_2_all"/>
     </widget>
     <addaction name="menu_1"/>
     <addaction name="menu_3"/>
     <addaction name="menu_1_2"/>
     <addaction name="action_clean_uart"/>
     <addaction name="action_clean_all"/>
    </widget>
    <addaction name="action_load_dump"/>
    <addaction name="action_rst"/>
    <addaction name="menu_2"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>запись измерений</string>
    </property>
    <addaction name="wr_all"/>
    <addaction name="wr_1"/>
    <addaction name="wr_2"/>
   </widget>
   <addaction name="menuPlotter"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_load_dump">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>загрузить дамп</string>
   </property>
  </action>
  <action name="action_rst">
   <property name="text">
    <string>сбросить модуль</string>
   </property>
  </action>
  <action name="wr_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>все каналы</string>
   </property>
  </action>
  <action name="wr_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>канал №1</string>
   </property>
  </action>
  <action name="wr_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>канал №2</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>закрыть</string>
   </property>
  </action>
  <action name="action_clean_uart">
   <property name="text">
    <string>ЮАРТ</string>
   </property>
  </action>
  <action name="action_clean_all">
   <property name="text">
    <string>полная очистка</string>
   </property>
  </action>
  <action name="action_clean1_today">
   <property name="text">
    <string>за сегодня</string>
   </property>
  </action>
  <action name="action_clean1_all">
   <property name="text">
    <string>за всё время</string>
   </property>
  </action>
  <action name="action_clean2_today">
   <property name="text">
    <string>за сегодня</string>
   </property>
  </action>
  <action name="action_clean2_all">
   <property name="text">
    <string>за всё время</string>
   </property>
  </action>
  <action name="action_clean1_2_today">
   <property name="text">
    <string>за сегодня</string>
   </property>
  </action>
  <action name="action_clean1_2_all">
   <property name="text">
    <string>за всё время</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
